modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-retrofit2/api.mustache

```diff
@@ -47,6 +47,12 @@ import okhttp3.MultipartBody
 
 {{/isMultipart}}
 {{/x-kotlin-multipart-import}}
+{{#operation}}
+{{#isResponseFile}}
+import okhttp3.ResponseBody
+
+{{/isResponseFile}}
+{{/operation}}
```

Comment:
Thanks for the fix, which has one limitation though. It only works if all operations have one and only one binary response. Otherwise, there will be multiple import of okhttp3.ResponseBody.

I'll see if I can come up with a better fix later with extensions.

Reply:
Yes, that is right. But at least it compiles and the compiler ignores the redundant lines. It was the easiest fix that we could apply by changing the template locally in our project.

Thanks for merging it!

Reply:
@wing328  given that you was reviewing the original PR - may I possibly ask you if the changes from the https://github.com/OpenAPITools/openapi-generator/issues/9682 could be somehow integrated in the lib itself?